packages/teams-js/test/public/media.spec.ts

```diff
@@ -439,6 +440,133 @@ describe('media', () => {
           } as DOMMessageEvent);
         });
       });
+
+      it('videoController notifyEventToHost should fail in default version of platform', async () => {
```

Comment:
Please write a test to have the callback tested too.

Reply:
The example is super helpful, thank you!

Reply:
If this is easy/straightforward we can do it as part of the merge, but otherwise can open a bug to have someone handle it asynchronously (ha!) from merging.

Reply:
Opened a bug item, will link to it in our teams git magic channel